build.yaml

```diff
@@ -1204,3 +1203,55 @@ job_type:
                    *package_jenkins_osx_yosemite_box ]
           }
       timeout: 120
+
+#-----------------------------------------------------------------------------#
+# View definitions below this point
+#-----------------------------------------------------------------------------#
+
+views:
+  trial:
+    description: 'All trial jobs'
+    regex: 'run_(?i)trial_.*.*'
```

Comment:
well... they are jobs that are executed directly with 'trial' vs 'run-<something>-tests'